lib/views/pages/login_signup/login_page.dart

```diff
@@ -566,4 +590,51 @@ class _LoginScreenState extends State<LoginPage> with TickerProviderStateMixin {
           ),
         ));
   }
+
+  _exceptionToast(String msg) {
+    Widget toast = Container(
+      padding: const EdgeInsets.symmetric(horizontal: 24.0, vertical: 14.0),
+      decoration: BoxDecoration(
+        borderRadius: BorderRadius.circular(25.0),
+        color: Colors.red,
+      ),
+      child: Row(
+        mainAxisSize: MainAxisSize.min,
+        children: [
+          Expanded(
+            child: Text(
+              msg,
+              style: TextStyle(fontSize: 15.0, color: Colors.white),
+              textAlign: TextAlign.center,
+            ),
+          ),
+        ],
+      ),
+    );
+
+    fToast.showToast(
+      child: toast,
+      gravity: ToastGravity.BOTTOM,
+      toastDuration: Duration(seconds: 5),
+    );
+  }
+
+  Future<void> checkAndSetUrl() async {
+    setState(() {
+      isUrlCalled = true;
+    });
+
+    try {
+      await http.get('${dropdownValue.toLowerCase()}://${urlController.text}/');
+
+      setAPIURL();
```

Comment:
setAPIURL() method not exists, throwing error.

<img width="1167" alt="Screenshot 1942-12-27 at 7 53 55 PM" src="https://user-images.githubusercontent.com/26805882/111642262-ef99d080-8823-11eb-98c9-598a3b48da2b.png">